src/Bitbucket.Net/Models/Core/Projects/RepositorySize.cs

```diff
@@ -0,0 +1,13 @@
+﻿using Newtonsoft.Json;
+
+namespace Bitbucket.Net.Models.Core.Projects
+{
+    public class RepositorySize
+    {
+        [JsonProperty("repository")]
+        public long SizeBytes { get; set; }
```

Comment:
Those are quite different property name and json property name

Reply:
Indeed, but those are the right ones :)